GPU/GPUTracking/Global/GPUChainTrackingDebugAndProfiling.cxx

```diff
@@ -295,7 +295,8 @@ void GPUChainTracking::SanityCheck()
 
 void GPUChainTracking::RunTPCClusterFilter(o2::tpc::ClusterNativeAccess* clusters, std::function<o2::tpc::ClusterNative*(size_t)> allocator, bool applyClusterCuts)
 {
-  GPUTPCClusterFilter clusterFilter(*clusters);
+  const uint8_t filterType = GetProcessingSettings().tpcApplyClusterFilterOnCPU;
+  GPUTPCClusterFilter clusterFilter(*clusters, filterType);
```

Comment:
Could you apply the passing of the filterType to the class also in GPUChainTrackingClusterizer.cxx

Reply:
I'm not sure where exactly. I find calling the filter here: https://github.com/AliceO2Group/AliceO2/blob/dev/GPU/GPUTracking/Global/GPUChainTrackingClusterizer.cxx#L1030
Or do you mean to add the check to call the filter also if the `tpcApplyClusterFilterOnCPU >0 || param().tpcCutTimeBin > 0`?

Reply:
ah, forget what I wrote, I was confused...